src/tests

```diff
@@ -45,7 +45,7 @@ p({"class":"foam.nanos.test.Test","passed":1,"failed":0,"id":"ValidateTest","las
 // Uncomment the line below when CRUNCH is mature enough to replace our current system.
-// p({"class":"foam.nanos.crunch.crunchtest.CapabilityTest","id":"CapabilityTest","description":"test capability and junctions related"})
+p({"class":"foam.nanos.crunch.crunchtest.CapabilityTest","id":"CapabilityTest","description":"test capability and junctions related"})
```

Comment:
Please remove the comment above.

Reply:
Actually I committed this change by accident. Should we enable this now or later?